WEB/Src/WindowsServer/WindowsServer.Tests/AzureWebAppRoleEnvironmentTelemetryInitializerTest.cs

```diff
@@ -4,10 +4,10 @@
     using System.Globalization;
     using Microsoft.ApplicationInsights.DataContracts;
     using Microsoft.ApplicationInsights.Extensibility.Implementation;
-#if !NETCORE
-    using Microsoft.ApplicationInsights.WindowsServer.Azure;
-    using Microsoft.ApplicationInsights.WindowsServer.Azure.Emulation;
-#endif
+//#if !NETCOREAPP
+//    using Microsoft.ApplicationInsights.WindowsServer.Azure;
+//    using Microsoft.ApplicationInsights.WindowsServer.Azure.Emulation;
+//#endif
```

Comment:
if this isn't necessary, perhaps it can be deleted?

Reply:
sure. let me update!